### PR TITLE
Fix/bounding box origin rescaling

### LINKED ISSUE #42: Fix bounding box origin on rescaling

### PR DESCRIPTION
**Pull request by GuillaumeLazar**
Close #42

### CODE REVIEW

File: src/model/BoundingBox.cpp [hunk 1]
```diff
@@ -28,10 +28,12 @@ BoundingBox::~BoundingBox()
...
 {
-    QRectF boundingRect = mParentSketchItem->getGraphicsItem()->boundingRect();
-    qreal marginAdjust = 10;
+    QRectF boundingRect = mParentSketchItem->getBounds();
+
+    // Uncomment that if you want a margin on boundingbox
+    //qreal marginAdjust = 10;
     //boundingRect.adjust(-marginAdjust, -marginAdjust, marginAdjust, marginAdjust);
-    setRect(boundingRect);
+    //setRect(boundingRect);
```
Comment (synapticvoid) [inline src/model/BoundingBox.cpp]: I think we can safely delete these lines, it was there only to see bezier control points and bounding box control points at the same time!

Comment (GuillaumeLazar) [inline src/model/BoundingBox.cpp]: See that I added a commentary before keeping commented code :)

```
 // Uncomment that if you want a margin on boundingbox
```

But ok I will delete these lines soon.

Comment (synapticvoid) [inline src/model/BoundingBox.cpp]: Yep I've seen it, that made it much easier to understand it while reading your PR!

**Comment by GuillaumeLazar** [comment]
Dead code removed, you could now accept the PR

**Pull request merged**